map/everywhere_search_params.hpp

```diff
@@ -15,7 +15,8 @@ namespace search
 struct EverywhereSearchParams
 {
   using OnResults =
-      std::function<void(Results const & results, std::vector<bool> const & isLocalAdsCustomer)>;
+      std::function<void(Results const & results, std::vector<bool> const & isLocalAdsCustomer,
```

Comment:
How about to create a single structure, say
```c++
ProductInfo
{
  bool m_isLocalAdsCustomer;
  float m_ugcRating;
};
```
and return a single vector of these structs, instead of one vector?

Reply:
да, хорошая идея

Reply:
это гораздо удобнее расширять